.github/actions/ci-setup/action.yaml

```diff
@@ -2,7 +2,7 @@ name: "CI Setup"
 inputs:
   node-version:
     description: "Node version"
-    default: 18.14.1
+    default: 18.17.1
```

Comment:
We tried that in the original PR. The template doesn't work if we don't do this minor bump, we would have to do a downgrade of `next` to the previous major version

I don't really think this is that big of a risk as it's only a minor bump, and not a major one. I'm gonna of course leave the decision upto you and the team, though.

Reply:
The only reason why I would push for the tool to be released now is so that we can announce it everywhere along with the 'new things that are introduced to Fuel in beta-5' and have it included in the quickstart docs everywhere.

Reply:
At this stage, we need to be sure.

Can you please confirm this with the FE team?

Reply:
On it

Reply:
Just wondering if we already know if this could be an issue. Waiting for the answer to approve it.

Reply:
@Torres-ssf Luiz has confirmed that things are still working fine for the frontend team